libkbfs/init.go

```diff
@@ -94,6 +98,18 @@ func GetDefaultMDServer(ctx Context) string {
 	}
 }
 
+// GetDefaultMetadataVersion returns the default metadata version per run mode.
+func GetDefaultMetadataVersion(ctx Context) MetadataVer {
+	switch ctx.GetRunMode() {
+	case libkb.StagingRunMode:
+		return SegregatedKeyBundlesVer
+	case libkb.ProductionRunMode:
+		return InitialExtraMetadataVer
+	default:
+		return InitialExtraMetadataVer
```

Comment:
Hmm. I didn't want to switch it over yet while we're still using v2 in prod. To run kbfsdocker I manually hardcoded to v3.